tests/pfcwd/test_pfcwd_timer_accuracy.py

```diff
@@ -358,7 +358,7 @@ def retrieve_timestamp(self, pattern):
                 logger.warning("Get timestamp: Unexpected syslog message format, syslog_msg {}".format(syslog_msg))
                 return int(0)
 
-            timestamp_ms = self.dut.shell("date -d {} +%s%3N".format(timestamp))['stdout']
+            timestamp_ms = self.dut.shell("date -d '{}' +%s%3N".format(timestamp))['stdout']
```

Comment:
Could you add a comment clarifying the expected format of timestamp? This will help ensure that the shell command processes it correctly

Reply:
Thanks for your suggestion. 
It's not about the timestamp format, it should be the common thing for the CMD input, we need always using the quotes to ensure that the entire string is treated as a single argument.
For this case, if we could ensure the timestamp including "year" in the future, I plan to use local python datetime.strptime to get the milliseconds instead of running the CMD on the remote device.